cli/foundation-models/system/finetune/token-classification/token-classification.sh

```diff
@@ -10,7 +10,7 @@ registry_name="azureml"
 
 compute_cluster="gpu-cluster-big"
 # if above compute cluster does not exist, create it with the following vm size
-compute_sku="Standard_ND40rs_v2"
+compute_sku="Standard_NC24rs_v3"
```

Comment:
The change is missing for text classification

Reply:
updated emotion-detection.sh